src/Date/Extra/I18n/I_pt_br.elm

```diff
@@ -158,16 +86,13 @@ make them same length as double digit days of month.
 -}
 dayOfMonthWithSuffix : Bool -> Int -> String
 dayOfMonthWithSuffix pad day =
-    let
-        value =
-            case day of
-                1 ->
-                    "1º"
-
-                _ ->
-                    toString day
-    in
-        if pad then
-            padLeft 4 ' ' value
-        else
-            value
+  let
+    value =
+      case day of
+        1 -> "1º"
+        _ -> toString day
+  in
+    if pad then
+      padLeft 4 ' ' value
```

Comment:
Hum… now I got it. Probably it should be 2 then. My bad. I'll send a PR for that ; )